src/pages/public/home/container.js

```diff
@@ -18,14 +18,15 @@ export const useContainer = () => {
       .collection(GENERAL_CONSTANTS.LANG)
       .doc(lang)
       .collection(GENERAL_CONSTANTS.NEWS)
-      .onSnapshot((resp) => {
+      .onSnapshot((resp) => {        
         const newsData = resp.docs.map((item) => {
           return {
             id: item.id,
             ...item.data(),
```

Comment:
Możesz to zwrócić po prostu ({id: item.id, ...item.data()}) bez returna. Tylko sprawdź czy na pewno musimy mapowac to w ten sposób, bo ID dostajemy bezpośrednio z serwera pod nazwą id.